FND/Places/NorthAmerica/USPostalServiceAddressesIndividuals.rdf

```diff
@@ -0,0 +1,2119 @@
+		<fibo-fnd-utl-av:commonDesignation>BRDGE</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BRG</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BRIDGE</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BRG</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Brook">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>brook</rdfs:label>
+		<skos:definition>small, natural freshwater stream</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BRK</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BROOK</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BRK</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Brooks">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>brooks</rdfs:label>
+		<skos:definition>multiple small, natural freshwater streams</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BROOKS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BRKS</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Burg">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>burg</rdfs:label>
+		<skos:definition>city or town (informal)</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BURG</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BG</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Burgs">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>burgs</rdfs:label>
+		<skos:definition>multiple cities or towns (informal)</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BURGS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BGS</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Bypass">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>bypass</rdfs:label>
+		<skos:definition>road passing around a town or its center to provide an alternative route for through traffic</skos:definition>
+		<fibo-fnd-utl-av:commonDesignation>BYP</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BYPA</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BYPAS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BYPASS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>BYS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>BYP</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Camp">
```

Comment:
Looks like you gave up supplying definitions from this point on.

Reply:
If I could find a definition on the postal service site then I used it.  If they didn't supply one, such as for these street suffixes, then I didn't make the attempt.  I tried to follow the standard to the degree possible, together with their other documentation on the USPS web site.